torch/testing/_internal/opinfo/core.py

```diff
@@ -680,6 +683,12 @@ class OpInfo:
     # information about which tests to skip
     skips: Tuple = tuple()
 
+    # skip the test for a device
+    skip_device: Tuple = tuple()
+
+    # enable the test for a device
+    enable_skipped_device: Tuple = tuple()
+
```

Comment:
The information is per op, why do we need to define two variables?

Reply:
Design is changed, enabled xpu test with xpu_op_db.yaml, no new fields are needed in OpInfo.